src/encoded/static/components/genome_browser.js

```diff
@@ -445,7 +445,13 @@ class GenomeBrowser extends React.Component {
 
     handleOnFocus() {
         this.setState({ showAutoSuggest: false });
-        getCoordinateData(`${this.context.location_href.split('/experiments/')[0]}/suggest/?genome=${this.state.genome}&q=${this.state.searchTerm}`, this.context.fetch).then((response) => {
+        let coordinateHref = '';
+        if (this.context.location_href.indexOf('/experiments') > -1) {
```

Comment:
If you look at the traffic, the search is performed twice. I am not sure why and it may be outside the scope of this ticket.

Is there ever a case where the type is not an experiment nor annotation?

Reply:
I think that looking into the search being performed twice might be a new ticket.

I thought there wasn't a case where the type is not an experiment nor annotation but I checked with @hitz and he said this was not a valid assumption, that the browser is already on the cart pages and could be on any number of search related pages. So I updated again to make the query more general so that it should work for any page.